pkg/server/endpoints/registration/handler_test.go

```diff
@@ -342,13 +342,11 @@ func (s *HandlerSuite) TestCreateEntryAndCreateEntryIfNotExists() {
 	}
 
 	verifyEntry := func(entry *common.RegistrationEntry) {
-		storedEntry, err := s.ds.FetchRegistrationEntry(context.Background(), &datastore.FetchRegistrationEntryRequest{
-			EntryId: entry.EntryId,
-		})
+		storedEntry, err := s.ds.FetchRegistrationEntry(context.Background(), entry.EntryId)
 		s.Require().NoError(err)
 		s.Require().NotNil(storedEntry)
-		s.T().Logf("actual=%+v expected=%+v", storedEntry.Entry, entry)
-		s.Require().True(proto.Equal(storedEntry.Entry, entry))
+		s.T().Logf("actual=%+v expected=%+v", storedEntry, entry)
+		s.Require().True(proto.Equal(storedEntry, entry))
```

Comment:
This test predates those helpers. If Brian wants to fix them up, that's fine but seems a little out of scope for this PR. Particularly, I don't find much value in maintaining the registration handler tests since it will be removed in the near future (maybe not 1.0, but certainly by 1.1).